src/apiCalls.js

```diff
@@ -43,4 +45,26 @@ const loadData = () => {
   })   
 }
 
-export { loadData, pageData };
+const addBooking = (date, roomNumber) => {
```

Comment:
You're using the native HTML input type=date. Check that the date parameter is what you're expecting. I was getting the date before the selected date because of the time zone. This article helped me find a solution: https://gomakethings.com/how-to-get-and-set-a-date-object-from-an-input-with-vanilla-javascript/

Reply:
k gonna fix all of these in a follow-up!